lib/hammer_cli_foreman/user.rb

```diff
@@ -26,7 +26,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
 
       output ListCommand.output_definition do
         field :admin, _("Admin"), Fields::Boolean
-        field :auth_source_internal, _("Authorized by"), Fields::Reference
+        field :auth_source_name, _("Authorized by")
```

Comment:
Could you please use `field nil, _("Authorized by"), Fields::SingleReference, :key => :auth_source` ?
It will enable for showing the auth source id when `--show-ids` is set.